src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -2167,6 +2169,68 @@ public void testSendReminderForFeedbackSessionParticularUsers() throws Exception
         
     }
     
+    public void testSendFeedbackSessionSubmissionConfirmationEmail() throws Exception {
+        // private method. no need to check for authentication.
+        
+        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("session1InCourse1");
+        String courseName = CoursesLogic.inst().getCourse(fs.getCourseId()).getName();
+        
+        ______TS("success: registered student");
+        
+        StudentAttributes studentToSendConfirmation = dataBundle.students.get("student5InCourse1");
+        
+        EmailWrapper emailSent = fsLogic.sendConfirmationEmailForSubmission(fs.getCourseId(), fs.getFeedbackSessionName(),
+                                                                            studentToSendConfirmation.googleId, null, null);
```

Comment:
This means the email will actually be sent if we use third party email service. Ideally it should not be. Does this method (or rather the entire of the `testSendFeedbackSessionSubmissionConfirmationEmail()`) follow another test method?